processor/servicegraphprocessor/processor_test.go

```diff
@@ -306,7 +306,7 @@ func verifyDuration(t *testing.T, m pmetric.Metric) {
 	assert.Equal(t, 1, dps.Len())
 
 	dp := dps.At(0)
-	assert.Equal(t, float64(1000), dp.Sum()) // Duration: 1sec
```

Comment:
It might be worthwhile to add some tests with the feature gate enabled to ensure we're getting the `ms` units when  we're supposed to.

Reply:
Added!